scripts/jenkins/ardana/ansible/roles/input_model_generator/vars/templates/interface/full-spread.yml

```diff
@@ -23,83 +23,71 @@ interface_models:
     network_interfaces:
       - network_groups:
           - MANAGEMENT
-      - network_groups:
-          - ARDANA
-      - forced_network_groups:
+        forced_network_groups:
           - EXTERNAL-API
+      - network_groups:
+          - CONF
```

Comment:
If you rename this here, please do it in all other templates, to maintain consistency.